lm_eval/api/task.py

```diff
@@ -773,7 +772,7 @@ def construct_requests(self, doc, ctx, **kwargs):
                         Instance(
                             request_type="loglikelihood",
                             doc=doc,
-                            arguments=("", "{}".format(choice)),
+                            arguments=("", " {}".format(choice)),
```

Comment:
This might just be making it too complicated, but maybe the added space here should be `target_delimiter` instead?

Reply:
Oh hm yeah you might be right. Maybe let’s pull this change into a different PR and figure out how to address it?

Reply:
actually I think we should just not have a preceding space. will revert

Reply:
Sounds good